docker-compose.yml

```diff
@@ -410,7 +411,7 @@ services:
       - INFINISPAN_URL=infinispan:${INFINISPAN_PORT}
       - INFINISPAN_CLIENT_INTELLIGENCE=BASIC
       - GENNY_KAFKA_URL=kafka:${KAFKA_PORT}
-      - GENNY_KEYCLOAK_URL=https://keycloak.gada.io/auth
+      - GENNY_KEYCLOAK_URL=https://keycloak-bali.gada.io
```

Comment:
as you have env_file settings above , you can read GENNY_KEYCLOAK_URL from ${ENV_FILE} by saving GENNY_KEYCLOAK_URL to genny.env

Reply:
Fixed